pkg/ocm/clusters.go

```diff
@@ -281,6 +294,19 @@ func (c *Client) CreateCluster(config Spec) (*cmv1.Cluster, error) {
 
 	clusterObject := cluster.Body()
 
+	logFwBuilder := BuildLogForwaders(config)
+
+	if logFwBuilder != nil {
+		logForwarder, err := logFwBuilder.Build()
+		if err != nil {
+			return clusterObject, err
+		}
+
+		if _, err := c.SetLogForwarder(clusterObject.ID(), logForwarder); err != nil {
```

Comment:
On day 1 (create) we should not use this endpoint. Instead, we should add the log forwarder to the cluster body being sent to the API

This endpoint is meant to be used only for day 2 operations